react/src/views/Pages/Upload.jsx

```diff
@@ -24,19 +115,119 @@ class UploadPage extends React.Component {
   }
 
   render() {
+    const { classes } = this.props;
     return (
       <div className = "main_upload">
         <React.Fragment>
         <CssBaseline />
           <Container fixed>
-            <Typography component="div" style={{ backgroundColor: 'white', height: '10vh', width: '50vh', paddingLeft: '5vh', paddingTop: '3vh' }} id="container">
+           
+            <GridContainer>
+              <GridItem xs={12} sm={12} md={6}>
+                <Card>
+                  <CardHeader color="rose" icon>
+                    <CardIcon color="rose">
+                      <MailOutline />
+                    </CardIcon>
+                    <h4 className={classes.cardIconTitle}>Upload VR Video</h4>
+                  </CardHeader>
+                  <CardBody>
+                    <form>
+                      {/* <CustomInput
+                        labelText="Enter VR Title "
+                        id="video_title"
+                        formControlProps={{
+                          fullWidth: true
+                        }}
+                        inputProps={{
+                          type: "email"
+                        }}
+                        value = {this.state.vrtitle}
+                        
+                        
+                      /> */}
+                      {/* <div>
+                        <input type="text" value={this.state.vrtitle} placeholder="Upload VR Vide" 
+                        onChange = {this.updateVRTitle}/>
+                      </div> */}
+                      <div>
+                        <TextField
+                          id="standard-basic"
+                          className={"VRTitle"}
+                          label="Type Your VR Title"
+                          margin="normal"
+                          value = {this.state.vrtitle}
+                          onChange = {this.updateVRTitle}
```

Comment:
Is there any way to prevent the default text from overlapping with what is typed?
![image](https://user-images.githubusercontent.com/36905285/69397395-dc600a80-0c9a-11ea-8d44-9adc26ef0cfa.png)